lib/kitchen/driver/gce.rb

```diff
@@ -440,13 +440,12 @@ def generate_server_name
 
       def create_disks(server_name)
         disks = []
-
         config[:disks].each do |disk_name, disk_config|
           unique_disk_name = "#{server_name}-#{disk_name}"
           if disk_config[:boot]
             disk = create_local_disk(unique_disk_name, disk_config)
             disks.unshift(disk)
-          elsif disk_config[:disk_type] == "local-ssd"
+          elsif (disk_config[:disk_type] == "local-ssd") | disk_config[:custom_image]
```

Comment:
*Content Warning: nit-picking*

<details><summary>Nit Picked for discussion purposes, probably OK to ignore</summary>
I would expect the double-pipe `||` "logical or" operator here to be lazy about evaluating the statement if the left-hand side is true. The single-pipe `|` "or" operator evaluates both sides always.

Did you have a preference for the always-or over the lazy-or?
</details>

Reply:
Thanks for pointing this out; I think that it was just a typo on my end.